src/goto-analyzer/static_verifier.cpp

```diff
@@ -156,67 +157,104 @@ static void static_verifier_console(
 {
   irep_idt last_function_id;
   irep_idt function_file;
-
-  for(const auto &result : results)
+  ui_message_handlert *ui_message_handler =
+    dynamic_cast<ui_message_handlert *>(&m.get_message_handler());
+  auto const status_to_string =
+    [](static_verifier_resultt::statust status) -> std::string {
+    switch(status)
+    {
+    case static_verifier_resultt::TRUE:
+      return "SUCCESS";
+    case static_verifier_resultt::FALSE:
+      return "FAILURE";
+    case static_verifier_resultt::BOTTOM:
+      return "SUCCESS";
+    case static_verifier_resultt::UNKNOWN:
+      return "UNKNOWN";
+    default:
+      INVARIANT(false, "result is one of TRUE, FALSE, BOTTOM or UNKNOWN");
+      return "";
```

Comment:
The `default` case should be unnecessary. We have compiler warnings enabled for switch/case that don't cover all values of an enum.

Reply:
I always feel a bit weird about that because my opinion is that while _in theory_ we should be able to rely on enums having only the values of the constants they're declared with, but in practice the standard doesn't actually guarantee that :(

But I suppose if we have a code convention that we'll never convert from integers to enum values this is fine as well.

Reply:
If we do those kinds of conversions we're probably in pretty big trouble. Ideally we'd actually have compiler warnings in place to really forbid those conversions.